### PR TITLE
Ab#6478 meta extras new tooltip design v2

### PR DESCRIPTION
**Pull request by matthewthomsonnz**
ADO card: ☑️ [AB#6478](https://dev.azure.com/S72/fefacba9-96b6-4af2-a53d-050ed453e13a/_workitems/edit/6478)

Elab notes/AC: 📃 Jacks comment in ADO card
- [x] Has this been discussed with Delivery Team? (a while ago now)

Designs: 
https://app.zeplin.io/project/5fdaa165fbacf584a2ce4f8b/screen/6142bb9c6a68af5099bd16fb

## Description of work
- Enables tooltips on relish components of meta-detail-extras buttons
- changes i18n 'my list' to  "Remove from My List"
- adds css to display buttons correctly with opacity and tooltip center alignment
- removes css that I can see is no longer applicable (please check I have been liberal)

Implements Jacks suggestions:
- Enabled trailer open without redirect on carousel and carousel pause/resume on open/close
- Enables focusin/focusout to make tooltips accessible
- Enables aria labels

### Affected Clients
 - All clients who use Kibble

## Checklist
- [x] CI tests are passing Github actions (inc. linting)
- [x] Key areas of the feature outlined for context and testing
- [x] If there are designs for this work are they noted here and in the ADO card 
- [ ] Design review
- [x] Have checked this at multiple screen resolutions and range of browsers
- [ ] Moved ADO card to Dev/done, checked link to Github, tagged "Review"
- [x] Updated changelog (if applicable)
- [x] I promise to document any new feature toggles/configurations in the appropriate documentation

### CODE REVIEW

File: site/styles/_variables.scss
Comment (PeculiarGoat) [inline site/styles/_variables.scss:254]: Can you swap these for CSS variables please.  All new variables should be ideally be CSS variables instead of Sass variables and definitely not Bootstrap variables.

File: site/styles/_buttons.scss
Comment (matthewthomsonnz) [reply site/styles/_buttons.scss]: - Confirmed and deleted.

(confirmed doesnt apply to carousel trailer, film detail trailer, and... the elusive bundle trailer lol)

File: site/styles/_meta-detail.scss
Comment (curtisadams28) [inline site/styles/_meta-detail.scss:131]: Isn't this padding redundant? It's already being set on the line above.

**Review by curtisadams28 — APPROVED** [review]
LGTM pending Jack's approval.

File: site/styles/_cta-buttons.scss
Comment (matthewthomsonnz) [reply site/styles/_cta-buttons.scss]: - reverted

File: site/styles/_cta-buttons.scss
Comment (matthewthomsonnz) [reply site/styles/_cta-buttons.scss]: - added css rgb color var as per slack chats. Other hovers in a future PR.

File: site/styles/_cta-buttons.scss
Comment (matthewthomsonnz) [reply site/styles/_cta-buttons.scss]: - moved

File: site/styles/_meta-detail.scss
Comment (matthewthomsonnz) [reply site/styles/_meta-detail.scss]: I think this is just something the linter moved lol removed

File: site/templates/common/cta_buttons.jet
Comment (matthewthomsonnz) [reply site/templates/common/cta_buttons.jet:43]: Only had graham reply but think thats good enough
- removed all trailerRedirect mentions

File: site/styles/_variables.scss
Comment (matthewthomsonnz) [reply site/styles/_variables.scss]: - changed to css vars

**Review by l0ud0gg — APPROVED** [review]
Ran it up / Looks all good to me. just jack to confirm hisbits

**Comment by matthewthomsonnz** [comment]
Looks good, except is there some context to the extra poster image sizes? My understanding was there is only 160x40 at all breakpoints?

**Review by PeculiarGoat — APPROVED** [review]
I've merged in develop, ran the linter, and made a couple of small tweaks to speed this up.  Happy with this PR now!

**Pull request merged**
